upload_files.py

```diff
@@ -0,0 +1,123 @@
+import argparse
+import os
+import json
+
+from core_data_modules.logging import Logger
+from storage.google_cloud import google_cloud_utils
+from storage.google_drive import drive_client_wrapper
+
+from src.lib import PipelineConfiguration
+
+Logger.set_project_name("IMAQAL")
+log = Logger(__name__)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Uploads output files")
+
+    parser.add_argument("user", help="User launching this program")
+    parser.add_argument("google_cloud_credentials_file_path", metavar="google-cloud-credentials-file-path",
+                        help="Path to a Google Cloud service account credentials file to use to access the "
+                             "credentials bucket")
+    parser.add_argument("pipeline_configuration_file_path", metavar="pipeline-configuration-file-path",
+                        help="Path to the pipeline configuration json file")
+    parser.add_argument("run_id", metavar="run-id",
+                        help="Identifier of this pipeline run")
+    parser.add_argument("memory_profile_file_path", metavar="memory-profile-file-path",
+                        help="Path to the memory profile log file to upload")
+    parser.add_argument("data_archive_file_path", metavar="data-archive-file-path",
+                        help="Path to the data archive file to upload"),
+    parser.add_argument("production_csv_input_path", metavar="production-csv-input-path",
+                        help="Path to a CSV file with raw message and demographic response, for use in "
+                             "radio show production"),
+    parser.add_argument("csv_by_message_input_path", metavar="csv-by-message-input-path",
```

Comment:
While we're here, we could rename this to mesage_csv_input_path. I think that reads better, and is consistent with the phrasing for the production CSV